pkg/manifest/anaconda_installer.go

```diff
@@ -135,8 +138,20 @@ func (p *AnacondaInstaller) getBuildPackages(Distro) []string {
 	packages := p.anacondaBootPackageSet()
 	packages = append(packages,
 		"rpm",
-		"lorax-templates-generic",
 	)
+	// XXX switch this one something
+	// - OSName -> no
+	// - Explicit
+	if p.UseRHELLoraxTemplates {
+		packages = append(packages,
+			"lorax-templates-rhel",
+		)
+	} else {
+		packages = append(packages,
+			"lorax-templates-generic",
+		)
+	}
```

Comment:
I think t his is fine as a pipeline option set from outside.  The distro sets the option on the image kind, and the image function passes it down to the appropriate pipeline(s).  It's how we do all our distro-level switches.

Reply:
Right, but in this case we have no distro/image function (yet).